connect.php

```diff
@@ -1,10 +1,9 @@
 <?php
-include './config.php';
 date_default_timezone_set('Asia/Calcutta');
 $host = 'localhost';
 $db   = 'infinitoAdmin';
 $user = 'root';
-$pass = DBPASSWORD;
```

Comment:
@Code-Aditya-14 Do not hardcore $pass as sql password will be different for everyone. So instead create a config.php file with DBPASSWORD as key to store sql password. 

Follow the docs in [contributing.md](https://github.com/Infinito-IIT-Patna/Infinito2020/blob/master/CONTRIBUTING.md)